HPXMLtoOpenStudio/resources/misc_loads.rb

```diff
@@ -27,7 +27,15 @@ def self.apply_plug(model, plug_load, obj_name, living_space)
     if plug_load.location == HPXML::LocationExterior
       # Set all heat gain as lost
       sens_frac = 0.0
-      lat_fract = 0.0
+      lat_frac = 0.0
+    end
+
+    if apply_ashrae140_assumptions
+      # ASHRAE 140, Table 7-9: Sensible load accounts for 0.822 of the total load and latent load accounts for 0.178. Sensible loads are 70% radiative and 30% convective.
+      rad_frac = 0.7 * 0.822
+      lat_frac = 0.178
```

Comment:
We are already specifying some of these values in HPXML file, right? So all you need here is `rad_frac = 0.7 * sens_frac` and can leave out `lat_frac` (let it pass through).